src/main/java/io/cryostat/rules/RuleProcessor.java

```diff
@@ -286,23 +286,26 @@ private void activate(Rule rule, ServiceRef serviceRef) {
                                                 this::archivalFailureHandler);
                                 Pair<ServiceRef, Rule> key = Pair.of(serviceRef, rule);
                                 Set<Long> ids = tasks.computeIfAbsent(key, k -> new HashSet<>());
+                                int initialDelay = rule.getInitialDelaySeconds();
+                                int archivalPeriodSeconds = rule.getArchivalPeriodSeconds();
+                                if (initialDelay <= 0) {
+                                    initialDelay = archivalPeriodSeconds;
+                                }
+                                if (rule.getPreservedArchives() <= 0
+                                        || initialDelay <= 0
```

Comment:
Since `initialDelay` that is `<= 0` will set to `archivalPeriodSeconds`. We can skip this expression `initialDelay <= 0`?